hw_diag/utilities/blockchain.py

```diff
@@ -11,7 +11,7 @@ def get_helium_blockchain_height():
     TypeError - if the key ['data']['height'] in response is not found.
     """
     result = requests.get('https://api.helium.io/v1/blocks/height',
-                            timeout=os.environ['DEFAULT_TIMEOUT'])
+                          timeout=os.environ['DEFAULT_TIMEOUT'])
```

Comment:
This will cause an error if there's no environment variable set. We want it to use a hardcoded default. Like [this](https://github.com/NebraLtd/hm-config/blob/16f925719060275ba7ebc39ac7d676ab99743b26/gatewayconfig/__main__.py#L25).

```
timeout=os.getenv('HELIUM_API_TIMEOUT_SECONDS', 5)
```